.teamcity/Gradle_Check/configurations/PerformanceTest.kt

```diff
@@ -26,11 +26,6 @@ class PerformanceTest(model: CIBuildModel, type: PerformanceTestType, stage: Sta
         }
     }
 
-    requirements {
-        // TODO this can be removed once https://github.com/gradle/gradle-private/issues/1861 is closed
-        doesNotContain("teamcity.agent.name", "ec2")
```

Comment:
Can we leave the coordinator job only running on our infrastructure? There are still some relocatability issues with its inputs and it is a long running job so I don't want to use spot instances for it. WDYT?